File: 21023rdlab/app/src/main/java/com/example/a2102_3rdlab/MainActivity.java
package com.example.a2102_3rdlab;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button buttonFirst = findViewById(R.id.buttonFirst);
        Button buttonSecond = findViewById(R.id.buttonSecond);
        Button buttonThird = findViewById(R.id.buttonThird);
        Button buttonExit = findViewById(R.id.buttonExit);

        buttonFirst.setOnClickListener(v -> startActivity(new Intent(MainActivity.this, FirstActivity.class)));
        buttonSecond.setOnClickListener(v -> startActivity(new Intent(MainActivity.this, SecondActivity.class)));
        buttonThird.setOnClickListener(v -> startActivity(new Intent(MainActivity.this, ThirdActivity.class)));
        buttonExit.setOnClickListener(v -> finish());
    }
}

File: 28043rdlabonkotlin/app/src/main/java/com/example/a2804_3rdlabonkotlin/MainActivity.kt
package com.example.a2804_3rdlabonkotlin

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.widget.Button

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val buttonFirst = findViewById<Button>(R.id.buttonFirst)
        val buttonSecond = findViewById<Button>(R.id.buttonSecond)
        val buttonThird = findViewById<Button>(R.id.buttonThird)
        val buttonExit = findViewById<Button>(R.id.buttonExit)

        buttonFirst.setOnClickListener { startActivity(Intent(this, FirstActivity::class.java)) }
        buttonSecond.setOnClickListener { startActivity(Intent(this, SecondActivity::class.java)) }
        buttonThird.setOnClickListener { startActivity(Intent(this, ThirdActivity::class.java)) }
        buttonExit.setOnClickListener { finish() }
    }
}